aiobiketrax/client.py

```diff
@@ -223,6 +227,11 @@ def _subscription(self) -> Optional[models.Subscription]:
         """
         return self._account._subscriptions.get(self._id)
 
+    @property
+    def _trip(self) -> Optional[models.Trip]:
```

Comment:
Suggestion: refactor to `_trips` and `trips`, then add a `latest_trip` to select the newest trip (by date).